internal/utils/hash_file.go

```diff
@@ -7,16 +7,23 @@ import (
 	"os"
 )
 
-func HashFile(path string) (string, error) {
+// HashFile returns the SHA256 hashsum of a file.
+// If prependHashType is true, the string "sha256:" is prepended
+func HashFile(path string, prependHashType bool) (string, error) {
```

Comment:
nit: may be better to have a `HashFile(path string)` and a separate `HashFileWithPrependedType` or something similar to make the code easier to read than `HashFile(foo, true/false)`.

Alternatively, we can just rename this function `SHA256Hash` or something similar, and if a caller wants to prepend something they can do it themselves since they've explciitly asked for sha256.

Reply:
sure, I'll do the second one